src/DependencyInjection/Configuration.php

```diff
@@ -121,6 +121,7 @@ private function addContextsSection(ArrayNodeDefinition $node)
                                         ->scalarNode('format')->defaultValue('jpg')->end()
                                         ->scalarNode('constraint')->defaultValue(true)->end()
                                         ->scalarNode('resizer')->defaultNull()->end()
+                                        ->arrayNode('resizer_options')->useAttributeAsKey('name')->prototype('scalar')->end()
```

Comment:
Default should be an empty array, right?

Reply:
Default should be the absence of array (in config).
Resizer, as i think, should get the empty array as default any time.

Another way - to predefine the lot of the keywords (like crop, background_color, watermark, crop_direction, but i didn't think that that will be enough for someone).

Another-another way - to make some system that can check all resizers and get allowed options from it - but i have enought skills to make it and it could be overkill.

Reply:
you about ->defaultValue([]) ?
i checked, it adds empty array anyway.